ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-pim-sparse-mode.j2

```diff
@@ -12,9 +12,7 @@
 | Rendezvous Point Address | Group Address |
 | ------------------------ | ------------- |
 {%             for rp_address in router_pim_sparse_mode.ipv4.rp_addresses | arista.avd.natural_sort %}
-{%                 for group in router_pim_sparse_mode.ipv4.rp_addresses[rp_address].groups | arista.avd.natural_sort %}
-| {{ rp_address }} | {{ group }} |
-{%                 endfor %}
+| {{ rp_address }} | {{ router_pim_sparse_mode.ipv4.rp_addresses[rp_address].groups | arista.avd.default(['-']) | join(', ') }} |
```

Comment:
For readability, we prefer to set a variable above when we need logic to define the value. Here we only have two columns, so it can work as is. But when we have wider tables it becomes much harder to read with inline logic.
I would prefer if you could set a `rp_groups` variable above.
Same under VRF.
Thanks

Reply:
Thanks for the advice, I made the changes as requested

Reply:
@gusmb Thank you. You forgot the `{{   }}` around the variable.